wire-grpc-server-generator/src/test/golden/unitService.kt

```diff
@@ -128,8 +128,9 @@ public object MyServiceWireGrpc {
   }
 
   public class BindableAdapter(
+    context: CoroutineContext = kotlin.coroutines.EmptyCoroutineContext,
     private val service: () -> MyServiceServer,
```

Comment:
This is a binary-breaking signature change for the JVM. You will need to generate `@JvmOverloads` on the constructor in order to retain compatibility.